Missing core_version.h when building using non-finalized platforms

As a result of [this change](https://github.com/tzapu/WiFiManager/commit/da02b59a21cff08f6e1c9858e06228622ecf9568#diff-71e2eff4cef6c58b55b7fa975c102042205487f444ac3a080c99de73280c12c6R68-R74), building in PlatformIO using upstreams that are not yet released versions results in an error.

Example of un-released upstream in platformio.ini:
```
[env:esp32_s2_wifi]
platform = https://github.com/platformio/platform-espressif32.git#feature/arduino-upstream
```

Error:
`.pio/libdeps/esp32_s2_wifi/WiFiManager/WiFiManager.h:69:14: fatal error: core_version.h: No such file or directory`


Using the feature/arduino-upstream platform is the recommended method for being able to compile for the ESP32-S2, amongst other chips.

--- a/WiFiManager.h
+++ b/WiFiManager.h
@@ -66,10 +66,10 @@
     // #pragma message "ESP_ARDUINO_VERSION_PATCH = " STRING(ESP_ARDUINO_VERSION_PATCH)
     #define VER_ARDUINO_STR STRING(ESP_ARDUINO_VERSION_MAJOR)  "."  STRING(ESP_ARDUINO_VERSION_MINOR)  "."  STRING(ESP_ARDUINO_VERSION_PATCH)
 #else
-    #include <core_version.h>
+    // #include <core_version.h>
     // #pragma message "ESP_ARDUINO_VERSION_GIT  = " STRING(ARDUINO_ESP32_GIT_VER)//  0x46d5afb1
     // #pragma message "ESP_ARDUINO_VERSION_DESC = " STRING(ARDUINO_ESP32_GIT_DESC) //  1.0.6
-    #define VER_ARDUINO_STR STRING(ARDUINO_ESP32_GIT_DESC)
+    #define VER_ARDUINO_STR "Unknown"
     // #pragma message "ESP_ARDUINO_VERSION_REL  = " STRING(ARDUINO_ESP32_RELEASE) //"1_0_6"
 #endif
 

